app/src/main/java/com/mealsmadeeasy/ui/home/profile/ProfileFragment.kt

```diff
@@ -0,0 +1,129 @@
+package com.mealsmadeeasy.ui.home.profile
+
+import android.app.AlertDialog
+import android.app.DatePickerDialog
+import android.content.Context
+import android.os.Bundle
+import android.view.LayoutInflater
+import android.view.View
+import android.view.ViewGroup
+import android.widget.*
+import com.mealsmadeeasy.R
+import com.mealsmadeeasy.model.Sex
+import com.mealsmadeeasy.ui.BaseFragment
+
+import java.util.*
+import android.widget.ArrayAdapter
+import java.text.SimpleDateFormat
+
+
+const val MIN_AGE = 13
+const val INCHES_IN_FEET = 12
+const val MIN_BMI = 15
+const val MAX_BMI = 50
+
+const val FEET_TO_METERS = 0.3048
+const val INCHES_TO_METERS = 0.0254
+const val POUNDS_TO_KILOGGRAMS = 0.453592
+
+class ProfileFragment : BaseFragment() {
+
+    companion object {
+        fun newInstance(): ProfileFragment = ProfileFragment()
+
+        fun calculateBMI(feet: Int, inches: Int, pounds: Int): Double {
+            return poundsToKilograms(pounds) / heightToMeters(feet, inches)
+        }
+
+        fun heightToMeters(feet: Int, inches: Int): Double {
+            return feet * FEET_TO_METERS + inches * INCHES_TO_METERS
+        }
+
+        fun poundsToKilograms(pounds: Int): Double {
+            return pounds * POUNDS_TO_KILOGGRAMS
+        }
+
+        fun heightToInches(feet: Int, inches: Int): Int {
+            return feet * INCHES_IN_FEET + inches
+        }
+    }
+
+    override fun onCreateView(inflater: LayoutInflater, container: ViewGroup?,
+                              savedInstanceState: Bundle?): View? {
+        val root = inflater.inflate(R.layout.view_user_profile, container, false)
+
+        // Populate sex spinner values
+        val sexSpinner = root.findViewById<Spinner>(R.id.sex_spinner)
+        val sexValues: List<String> = listOf(
+                Sex.MALE.toString(), Sex.FEMALE.toString(), Sex.OTHER.toString(), getString(R.string.profile_spinner_prompt))
+        val sexAdapter = HintAdapter(this.activity, android.R.layout.simple_list_item_1, sexValues)
+        sexAdapter.setDropDownViewResource(android.R.layout.simple_spinner_dropdown_item)
+        sexSpinner.adapter = sexAdapter
+        sexSpinner.setSelection(sexAdapter.count)
+
+        // Set up birthday date picker
+        val dateText = root.findViewById<EditText>(R.id.birthday_edit_text)
+        val cal = Calendar.getInstance()
```

Comment:
It's in Java's util class, but other than that I don't know. Its convenient since there's easy conversion of year, month, day into ints which DateTime things require.